api_app/services/logging.py

```diff
@@ -55,6 +70,7 @@ def initialize_logging(logging_level: int, correlation_id: str = None) -> loggin
     logger = logging.getLogger()
 
     try:
+        logger.addFilter(ExceptionTracebackFilter())
```

Comment:
This this impact only the logs sent to AppInsights or the ones printed to the console too?